test/README.md

```diff
@@ -303,6 +303,15 @@ Use the `-v` option for verbose output.
 
 #### Dependencies
 
+##### Shell
+
+Several lint tests require a `bash` shell version 4 or higher to be available.
+
+Especially modern macOS versions (at least 10.15) still require installation of
```

Comment:
I think the additions here could be condensed into:

```diff

diff --git a/test/README.md b/test/README.md
index 87a3acb76..cb96697e4 100644
--- a/test/README.md
+++ b/test/README.md
@@ -303,6 +303,13 @@ Use the `-v` option for verbose output.
 
 #### Dependencies
 
+##### Bash
+
+Several lint tests require a `bash` shell version 4 or higher.
+macOS systems require upgrading the system provided `bash`.
+
+##### Python
+
```

Reply:
done, thanks!